auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -66,7 +66,7 @@
             :items="columns"
             :menu-props="{ bottom: true, offsetY: true }"
           >
-            <template v-slot:selection="{ item }"></template>
+            <template v-slot:selection></template>
```

Comment:
@shabeeb-aot let me know if this is ok.

Reply:
I just verified in Dev and, yes, the drop-down  menu for Columns to Show displays all the items. Eg:

![Uploading image.png…]()